doc/writing-puzzles.md

```diff
@@ -75,70 +70,68 @@ Your first step will be to make a "sandwich" directory somewhere.
 
     $ mkdir sandwich
     $ cd sandwich
-    $
```

Comment:
I put these here to illustrate that there would be no output. Obviously you wouldn't expect output from `cd` but other times it's not so obvious. I'm not married to this typographic practice but I want to make sure you've at least considered the reasoning before I accept this diff.

#TypographicPedantry